includes/codegen/QCodeGenBase.class.php

```diff
@@ -815,54 +815,62 @@ public function ModelConnectorLabelVariableName($objColumn) {
 		 *
 		 * @param QColumn|QReverseReference|QManyToManyReference $objColumn
 		 *
-		 * @return string Class name of control which can handle this column's data
+		 * @return AbstractControl_CodeGenerator helper object
 		 * @throws Exception
 		 */
-		public function ModelConnectorControlClass($objColumn) {
-
+		public function GetControlCodeGenerator($objColumn) {
```

Comment:
You removed ModelConnectorControlClass and replaced it with this. This is more difficult to override in QCodeGen.php and specify a default class for each table type. I would prefer you to reinstate ModelConnectorControlClass, and then base this function on the results of that function. That would make it much easier for a developer to specify default classes for data types.